src/NodaTime/project.json

```diff
@@ -13,6 +13,12 @@
     "requireLicenseAcceptance": false,
     "repository": {
       "url": "https://github.com/nodatime/nodatime.git"
+    },
+    "files": {
+      "mappings": {
+        "build/": "../../packaging/NodaTime.targets",
+        "lib/native/": [ "../../packaging/_._" ]
```

Comment:
Okay. This is highly weird black magic, IMO - I hate the idea that every library author needs to include this due to one awkward language. I hope it gets fixed in later tooling.